src/Compilers/VisualBasic/vbc/Program.cs

```diff
@@ -1,6 +1,7 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
 using Microsoft.CodeAnalysis.BuildTasks;
+using System;
```

Comment:
@pharring if only the IDE would sort my usings when I add them with the light bulb ... ;)